platforms/android/tangram/src/main/java/com/mapzen/tangram/NativeLibraryLoader.java

```diff
@@ -0,0 +1,16 @@
+package com.mapzen.tangram;
+
+public class NativeLibraryLoader {
+    public static final boolean sNativeLibraryLoaded = loadLibrary();
```

Comment:
This is .. possibly too clever. It works as long as we're very careful to only reference this class in one spot, but it makes me nervous.